_emails/us/california/fresno.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Fresno, CA"
+permalink: "/fresno"
+name: "Letter to City Council"
+city: "Fresno"
+state: "CA"
+layout: "email"
+recipients:
+- "esmeralda.soria@fresno.gov"
+- "miguel.arias@fresno.gov"
+- "paul.capriogolio@fresno.gov"
+- "mike.karbassi@fresno.gov"
+- "luis.chavez@fresno.gov"
+- "garry.bredefeld@fresno.gov"
+- "nelson.esparza@fresno.gov"
+- "wilma.quan@fresno.gov"
+- "yvonne.spence@fresno.gov"
+subject: "[Add Unique Subject Line Here!]"
+body: |-
+  Hello City Council Member,
+  
+  My name is [YOUR NAME] and I am a resident of Fresno, CA. As you may well know this year, city council officials are expecting a budget deficit of $32 million due to Covid-19.
+  
+  The Fresno Police Department makes up approximately 53% of the city's budget, more than the budgets of FAX, PARCS, Human Services, Affordable Housing, and economic development COMBINED. In light of recent police violence, as well as documented cases of FPD abuse and racial profiling, I urge you to advocate for reducing funding to FPD during the City of Fresno budget hearing on Monday, June 15th, 2020. I would like to redirect funding away from FPD and into health, housing, education, and social services that support vulnerable residents and residents of color, especially lower-income Black residents. I would also urge you in your future voting patterns to advocate for an eventual abolition of the Fresno Police Department in favor of investing into non-violent social services, restorative justice services, and investing back into minority communities.
+  
+  Thank you for your time,
+  
+  [YOUR NAME]
```

Comment:
This should include all aspects of the signature. Please see email style guide.

Reply:
my bad - just updated